--- /dev/null
+++ ../src/MySQLImport.php
@@ -38,7 +38,7 @@
 	 * @param  string filename
 	 * @return int
 	 */
-	public function load($file)
+    public function load($file)
 	{
 		$handle = strcasecmp(substr($file, -3), '.gz') ? fopen($file, 'rb') : gzopen($file, 'rb');
 		if (!$handle) {
@@ -73,9 +73,13 @@
 
 			} elseif (substr($ts = rtrim($s), -strlen($delimiter)) === $delimiter) {
 				$sql .= substr($ts, 0, -strlen($delimiter));
-				if (!$this->connection->query($sql)) {
-					throw new Exception($this->connection->error . ': ' . $sql);
-				}
+                try {
+                    if (!$this->connection->query($sql)) {
+                        throw new Exception($this->connection->error . ': ' . $sql);
+                    }
+                } catch (mysqli_sql_exception $mysqli_sql_exception) {
+                    throw new Exception($mysqli_sql_exception->getMessage() . ': ' . $sql);
+                }
 				$sql = '';
 				$count++;
 				if ($this->onProgress) {
